### PR TITLE
Settings UI changes

### PR DESCRIPTION
**Pull request by biafra23**
Handles #1201

Changes proposed in this pull request:
- Change header font size and letter spacing (scope change: All headers in all setting s and tx list should be changed)
- Add help link for Fallback handler
- Contract version -> Safe version

Before | After
-------|------
Screenshot_20210203-151230 | safe_settings_after
Screenshot_20210203-151248 | safe_aettings_advanced_after
Screenshot_20210203-151235 | app_asettigns_advanced_after
Screenshot_20210203-152429 | queue_after
Screenshot_20210203-152443 | history_after

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -243,6 +243,8 @@
     <string name="safe_settings_fallback_handler">Fallback handler</string>
     <string name="safe_settings_nonce">Nonce</string>
     <string name="safe_settings_modules">Addresses of enabled modules</string>
+    <string name="safe_settings_fallback_handler_help">What is a fallback handler and how does it relate to the Gnosis Safe?</string>
+    <string name="safe_settings_fallback_handler_help_url">https://help.gnosis-safe.io/en/articles/4738352-what-is-a-fallback-handler-and-how-does-it-relate-to-the-gnosis-safe</string>
```
Comment (elgatovital) [inline app/src/main/res/values/strings.xml:247]: this could go to static strings

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -127,11 +128,11 @@
         <item name="android:layout_height">@dimen/header_height</item>
         <item name="android:gravity">center_vertical</item>
         <item name="android:paddingLeft">16dp</item>
-        <item name="android:textSize">10sp</item>
+        <item name="android:textSize">13sp</item>
```
Comment (elgatovital) [inline app/src/main/res/values/styles.xml:131]: This will affect not only settings.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (biafra23) [reply app/src/main/res/values/strings.xml]: I moved both urls. See: a2660a78fd7fbf9914474fcff36cc46429e12f58

**Review by elgatovital — APPROVED** [review]

**Comment by elgatovital** [comment]
:+1:

**Pull request merged**